google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java

```diff
@@ -78,7 +74,7 @@ public void failed(Subscriber.State from, Throwable failure) {
             // Handle error.
           }
         },
-        executor);
+        Executors.newCachedThreadPool());
```

Comment:
I think we also need to shutdown this executor.

Reply:
Done